README.md

```diff
@@ -120,11 +120,51 @@ luatest.helpers.retrying({timeout = 1, delay = 0.1}, failing_function, arg1, arg
 luatest.helpers.retrying({}, function() server:http_request('get', '/status') end)
 ```
 
+## List of luatest functions
+
+| Assertiions |  |
+| :--- | --- |
+| `assert (value[, message])` | Check that value is truthy. |
+| `assert_almost_equals (actual, expected, margin[, message])` | Check that two floats are close by margin. |
+| `assert_covers (actual, expected[, message])` | Checks that map contains the other one. |
+| `assert_equals (actual, expected[, message[, deep_analysis]])` | Check that two values are equal. |
```

Comment:
Are these correct asserts? I am using `t.assertEquals` in my tests.

Reply:
see previous conversation